demos/nodejs/demo-app/vite.config.mts

```diff
@@ -1,89 +1,84 @@
-import wasm from 'vite-plugin-wasm';
-import topLevelAwait from 'vite-plugin-top-level-await';
-import { fileURLToPath, URL } from 'url';
+import wasm from "vite-plugin-wasm";
```

Comment:
I think we might need to add a `.prettierrc` file to this demo

Reply:
Done; the diff is much better now